src/styles.scss

```
@@ -71,6 +71,28 @@ b { font-weight: 500; }
     color: inherit;
 }
 
+/* GFM tables rendered by <remark>. ngx-remark ships no CSS and emits a bare
+   <table>, which the browser draws without borders. Scoped to `remark` so it
+   cannot reach Angular Material's tables. Column alignment from the Markdown
+   (`:---:`) is applied inline by ngx-remark and so still wins over this. */
+remark table {
```

Stylelint (17.14.0)

[error] Unknown type selector "remark" (selector-type-no-unknown)

(selector-type-no-unknown)